requirements/common.txt

```diff
@@ -310,3 +310,16 @@ iso8601==0.1.12 \
 
 enum34==1.1.6; python_version < '3' \
     --hash=sha256:6bd0f6ad48ec2aa117d3d141940d484deccda84d4fcd884f5c3d93c23ecd8c79
+
+python-jose==2.0.0 \
```

Comment:
Looking at the Auth0 docs they say to use [python-jose-cryptodome](https://github.com/capless/python-jose-cryptodome) rather than [python-jose](https://github.com/mpdavis/python-jose) (https://auth0.com/docs/quickstart/backend/python#install-the-dependencies).

However their recommendation doesn't have much GitHub activity and appears to just be a fork of python-jose that changed the vulnerable pycrypto dependency for pycryptodome ([blog post about the vulnerability]()), but (a) hasn't kept up with python-jose, (b) is now redundant since python-jose has just switched to pycryptodome too (https://github.com/mpdavis/python-jose/commit/98406bc02bb42e0a7373307cb7aad44293086d1a).

Also looking at https://jwt.io/#libraries-io there appears to be yet another JWT option that's much more popular/active than python-jose:
https://github.com/jpadilla/pyjwt/

As such, I think it would be good to:
1. Double check which of python-jose and PyJWT is best for our use-case/long term reliability
2. At some point file an issue against the Auth0 docs getting them to recommend python-jose and not the stale python-jose-pycryptodome package

Reply:
Created https://github.com/auth0/docs/issues/5611.

Reply:
Still no answer. I will leave it as `python-jose` (recommended by auth0) until I do hear back.

Reply:
Could you bump this dep to the newly release `2.0.2`? (https://pypi.python.org/pypi/python-jose)